utils/systemd/daos-agent.service

```diff
@@ -14,3 +14,6 @@ ExecStart=/usr/bin/daos_agent -i
 StandardOutput=journal
 StandardError=journal
 Restart=always
+
+[Install]
+WantedBy = multi-user.target
```

Comment:
Is `multi-user.target` the right target?

Reply:
It should be. That is the default target for runlevels 3 4 and 5.

Reply:
sorry its 2 3 and 4 not 3 4 and 5